.github/workflows/try-runtime.yml

```diff
@@ -0,0 +1,66 @@
+name: Try-runtime
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+env:
+  TRY_RUNTIME_CLI_VERSION_TAG: v0.7.0
+  CARGO_HOME: ./.cargo
+  SUBCOMMAND_EXTRA_ARGS: "--no-weight-warnings"
```

Comment:
Why are we ignoring the weight warnings?

Reply:
Removed